packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/FLTCam.m

```diff
@@ -88,10 +85,10 @@ @interface FLTCam () <AVCaptureVideoDataOutputSampleBufferDelegate,
 @property(assign, nonatomic) int maxStreamingPendingFramesCount;
 
 @property(assign, nonatomic) UIDeviceOrientation lockedCaptureOrientation;
-@property(assign, nonatomic) CMTime lastVideoSampleTime;
-@property(assign, nonatomic) CMTime lastAudioSampleTime;
-@property(assign, nonatomic) CMTime videoTimeOffset;
-@property(assign, nonatomic) CMTime audioTimeOffset;
+@property(assign, nonatomic) CMTime recordingTimeOffset;
+@property(assign, nonatomic) CMTime lastSampleEndTime;
+@property(nonatomic) AVCaptureOutput *outputForOffsetAdjusting;
```

Comment:
we didn't have this output. can this be just a local variable?

Reply:
That output is either audio or video output depending on whether audio or video output should be used for offset adjusting.